src/hooks/useChat.test.tsx

```diff
@@ -20,33 +21,38 @@ describe("useChat", () => {
     jest.restoreAllMocks();
   });
 
-  it("should initialize with correct defaults", () => {
-    const { result } = renderHook(() => useChat());
-    expect(result.current.state).toBe("idle");
-    expect(result.current.currentChat).toBeNull();
-    expect(result.current.chatHistory).toEqual([]);
+  it("should initialize with correct defaults", async () => {
+    const { result } = renderHook(() => useChat(), {
+      wrapper: createWrapper(),
+    });
+    await waitFor(() => {
+      expect(result.current.state).toBe("idle");
+      expect(result.current.currentChat).toBeNull();
+      expect(result.current.chatHistory).toEqual([]);
+    });
   });
 
-  it("send message", async () => {
+  it("can send messages", async () => {
     const message = "Test message";
-    const { result } = renderHook(() => useChat());
+    const { result } = renderHook(() => useChat(), {
+      wrapper: createWrapper(),
+    });
     act(() => {
       result.current.sendMessage(message);
     });
-    expect(result.current.state).toBe("waiting");
+    expect(result.current.state).toBe("loading");
 
-    act(() => {
-      waitFor(() => {
-        expect(result.current.state).toBe("idle");
-        expect(result.current.chatHistory).toEqual([
-          { role: "user", content: message },
-          { role: "assistant", content: "Test response" },
-        ]);
-      });
+    await waitFor(() => {
+      expect(result.current.state).toBe("success");
+      expect(result.current.chatID).toBe(42);
+      expect(result.current.chatHistory).toEqual([
+        { role: "human", content: message },
+        { role: "AI", content: "Test response" },
```

Comment:
The "ai" returned by Langchain is lowercase.

Reply:
Thanks!